prime/script/deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ ! -f prime/script/deploy.sh ]; then
+    echo "Run this script from project root dir (ostelco-core)"
```

Comment:
Pedantic comment, but use '(>&2 echo "Run this script ...")'.  The rationale being:
a) Error message to stderr
b) Do it in a subshell to avoid interacting with other redirects, if they are in action.